frame/support/test/tests/pallet.rs

```diff
@@ -217,6 +217,19 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type DoubleMap2<T> = StorageDoubleMap<_, Twox64Concat, u16, Blake2_128Concat, u32, u64>;
 
+	#[pallet::storage]
+	pub type NMap<T> = StorageNMap<_, storage::Key<Blake2_128Concat, u8>, u32>;
+
+	#[pallet::storage]
+	pub type NMap2<T> = StorageNMap<
+		_,
+		(
+			storage::Key<Twox64Concat, u16>,
```

Comment:
I am curious where is the `storage` imported from ? I can't find it in frame_support::pallet_prelude.

Reply:
It is imported from `pallet_prelude` actually:

```rust
pub mod pallet_prelude {
	// ...
	pub use crate::{
		EqNoBound, PartialEqNoBound, RuntimeDebugNoBound, DebugNoBound, CloneNoBound, Twox256,
		Twox128, Blake2_256, Blake2_128, Identity, Twox64Concat, Blake2_128Concat, ensure,
		RuntimeDebug, storage, // <----------------------------------------------------- here
		// ...
	}
}
```

Reply:
ah thanks